src/snowflake/snowpark/modin/plugin/docstrings/dataframe.py

```diff
@@ -1892,11 +1892,222 @@ def ne():
     def nlargest():
         """
         Return the first `n` rows ordered by `columns` in descending order.
+
+        Return the first `n` rows with the largest values in `columns`, in
+        descending order. The columns that are not specified are returned as
+        well, but not used for ordering.
+
+        This method is equivalent to
+        ``df.sort_values(columns, ascending=False).head(n)``, but more
```

Comment:
is it more performant? I would expect the SQL to be about the same for both.

Reply:
right it's not. This got copied from pandas doc. Removed now.

Reply:
In fact, depends on how the implementation look like, it might be the case, for example i would expect snowflake top n performs better than order by <> limit n.  however, based on the current implementation, difficult to tell. Safe to do not mention this for now